tests/integration/install/test_install.py

```diff
@@ -220,6 +220,43 @@ async def test_installs_and_runs_local_bladerunner(bladerunner_artifact):
             assert transpiled == sql_code
 
 
+async def test_installs_and_runs_pypi_bladerunner():
+    # Note: This test currently uses the user's home-directory, and doesn't really test the install process if the
+    # transpiler is already installed there: many install paths are a no-op if the transpiler is already installed.
```

Comment:
This test is different to those where I attached this comment, so the comment is misleading: this test is never a no-op because it always overwrites whatever the user previously had in their home-directory.